docs/reference/settings/health-diagnostic-settings.asciidoc

```diff
@@ -35,3 +35,13 @@ cluster wide health aspects such as disk space.
 `health.reporting.local.monitor.interval`::
 (<<cluster-update-settings,Dynamic>>) Determines the interval in which each node of the cluster monitors aspects that
 comprise its local health such as its disk usage.
+
+`health.ilm.max_time_on_action`::
+(<<cluster-update-settings,Dynamic>>) The minimum amount of time an index has to be in an {ilm-init} action before it is considered stagnant. Defaults to `1d` (1 day).
```

Comment:
It's good practice the first time we use an acronym in a page to spell it out, this will ensure that in every page the reader can find the reference and they don't have to start looking around. For example: adding `{ilm} ({ilm-init})` here is enough.